python/src/GeneralLinearModelResult_doc.i.in

```diff
@@ -11,11 +11,11 @@ residuals : :class:`~openturns.Point`
     The residual errors.
 relativeErrors : :class:`~openturns.Point`
     The relative errors.
-basis : collection of :class:`~openturns.Basis`
-    Collection of the  :math:`p` functional basis: :math:`(\varphi_j^l: \Rset^d \rightarrow \Rset)_{1 \leq j \leq n_l}` for each :math:`l \in [1, p]`.
+basis :  :class:`~openturns.Basis`
+    Collection of the  :math:`b` functional basis: :math:`(\varphi^l: \Rset^d \rightarrow \Rset^p)` for each :math:`l \in [1, b]`.
```

Comment:
I think we forgot to mention the api changes in the changelog, could you add it ?